ui/admin-portal/src/app/components/candidates/view/shareable-docs/shareable-docs.component.ts

```diff
@@ -76,6 +77,18 @@ export class ShareableDocsComponent extends AutoSaveComponentBase implements OnI
         this.candidate.listShareableDoc = null;
       }
       this.updatedShareableCV.emit(this.candidate.listShareableCv);
+    } else {
```

Comment:
I'm sure there's a much more elegant way to refactor this bulky method of code... it's just not coming to me atm and I want to get this pull request out. Any suggestion greatly appreciated!!